config/filter.d/nginx-http-auth.conf

```diff
@@ -4,7 +4,8 @@
 [Definition]
 
 
-failregex = ^ \[error\] \d+#\d+: \*\d+ user "\S+":? (password mismatch|was not found in ".*"), client: <HOST>, server: \S*, request: "\S+ \S+ HTTP/\d+\.\d+", host: "\S+"(, referrer: "\S+")?\s*$
+failregex = ^ \[error\] \d+#\d+: \*\d+ user \S*:? (password mismatch(: .*)?|(was )?not found(:| in) .*), client: <HOST>, server: \S*, request: "\S+ \S+ HTTP/\d+\.\d+", host: "\S+"(, referrer: "\S+")?\s*$
```

Comment:
I'm not sure that introducing of such changes is good at all. I mean for example this greedy catch-all here...
It is although not vulnerable, but will be slower thus.
If that would be optional...

I mean someone that need this (catch self generated message) can for example very easy add such regex within jail:
```
[nginx-http-auth]
failregex = %(known/failregex)s
            ^ \[error\] \d+#\d+: \*\d+ user \S*:? (password mismatch(: .*)? ...
```

@fail2ban/contributors, @yarikoptic Any thoughts about this PR?